core/sql/comexe/LateBindInfo.cpp

```diff
@@ -1319,37 +1049,102 @@ Int16 AnsiOrNskName::quoteNSKExtName()
     return 0;
 }
 
-void ResolvedNameListPre1800::translateFromNewVersion(ResolvedNameList *newrnl)
+///////////////////////////////////////////////////////////////////
+// class TrafSimilarityTableInfo
+///////////////////////////////////////////////////////////////////
+TrafSimilarityTableInfo::TrafSimilarityTableInfo(char * tableName,
+                                                 NABoolean isHive,
+                                                 char * hdfsRootDir, 
+                                                 Int64 modTS, Int32 numPartnLevels,
+                                                 Queue * hdfsDirsToCheck,
+                                                 char * hdfsHostName,
+                                                 Int32 hdfsPort)
+     : NAVersionedObject(-1),
+       tableName_(tableName),
+       hdfsRootDir_(hdfsRootDir),
+       modTS_(modTS), numPartnLevels_(numPartnLevels),
+       hdfsDirsToCheck_(hdfsDirsToCheck),
+       hdfsHostName_(hdfsHostName), hdfsPort_(hdfsPort)
```

Comment:
Looks like we need an initializer to set flags_ = 0. If we don't do this, and isHive is false, the HIVE flag will be uninitialized (and therefore random).